.gitignore

```diff
@@ -69,6 +69,7 @@ ssh.config
 
 # build tooling
 build.assets/tooling/bin/** 
+build.assets/.cache/
```

Comment:
This was an intentional change that's on other branches but never made it to v9 and makes switching branches a pain.

Technically not related to this PR, but so minor I'd rather include it than have to wait for another set of v9 tests to pass to merge it in.